### PR TITLE
Treat amount and fees as u64 (positive integers)

### PR DESCRIPTION
**Pull request by ok300**
Fixes https://github.com/breez/c-breez/issues/400

### CODE REVIEW

File: libs/sdk-core/src/models.rs
Comment (roeierez) [inline libs/sdk-core/src/models.rs:263]: existing wallets will fail to load probably because the fee_msat is stored as negative value in the db.
We need an SQL migration for that.

**Review by roeierez — CHANGES_REQUESTED** [review]

File: libs/sdk-core/src/models.rs
Comment (ok300) [reply libs/sdk-core/src/models.rs:263]: Done, Erdem helped with that.

**Comment by ok300** [comment]
> @ok300 is this ready for review? Asking because it is still marked as WIP.

@roeierez wanted to first test the new version myself, locally with c-breez. Just now finished my tests, ready for review.

Erdem already reviewed and said, from his side it's ready for merge.

**Comment by ok300** [comment]
FYI the scenarios I tested are:
* new Android image, restore previous wallet seed
  * result: ok (full history shown, incoming and outgoing payments)
* previous master build on an Android image, keep image but install new apk (of this PR), re-open wallet
  * result: ok (same as above)
* with this apk version, make outgoing payment (negative entries in history)
  * result: ok (new negative entries appeared as expected in history)

**Review by roeierez — APPROVED** [review]
LGTM! once CI passes can merge.

**Pull request merged**
